server/web/api/route.go

```diff
@@ -50,4 +52,7 @@ func SetupRoute(route gin.IRouter) {
 	}
 
 	authorized.GET("/ffp/:hash/:id", ffp)
+
+	authorized.GET("/httpfs", func(c *gin.Context) { c.Redirect(http.StatusFound, "/httpfs/") })
+	authorized.GET("/httpfs/*path", httpfs.HandleHttpfs)
```

Comment:
```suggestion
	authorized.HEAD("/httpfs/*path", httpfs.HandleHttpfs)
	authorized.GET("/httpfs/*path", httpfs.HandleHttpfs)
```
This fixes `rclone mount` not showing files.

Reply:
@ftk  perfect, this fixes rclone mount issue :)